### PR TITLE
Translation of Available methods section

### PR DESCRIPTION
**Pull request by doamaral**
Translation to Portuguese of the 'Available methods' section and its summary

close https://github.com/wlsf82/protractor-helper/issues/80

## Pull Request Checklist

- [x] Have you followed the steps in our [Contributing](https://github.com/wlsf82/protractor-helper/blob/master/docs/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/wlsf82/protractor-helper/pulls) for the same update/change?
- [-] Have you successfully ran tests with your changes locally?
- [-] Have you written new tests for your changes, if necessary?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you made improvements to the documentation, if necessary? ([docs/](https://github.com/wlsf82/protractor-helper/tree/master/docs) folder and [README.md](https://github.com/wlsf82/protractor-helper/blob/master/README.md))
- [x] Have you linked the correct [GitHub issue](https://github.com/wlsf82/protractor-helper/issues), if there is any?

> Pull requests that do not pass the coding static analysis and the automated tests on [Github Actions](https://github.com/wlsf82/protractor-helper/actions), and the code quality verification on [Better Code Hub](https://bettercodehub.com/) will not be reviewed.

### CODE REVIEW

**Comment by doamaral** [comment]
That's a big section. Some words are quite hard to translate to portuguese without being verbose.

File: README_PT-BR.md [hunk 1]
```diff
@@ -13,6 +13,33 @@ Muitas das funções auxiliares neste módulo usam a classe `protractor.Expected
   - [Exemplo utilizando o protractor-helper](#exemplo-utilizando-o-protractor-helper)
 - [Instalação](#instalação)
 - [Como usar e exemplos](#como-usar-e-exemplos)
+- [Helpers disponíveis](#helpers-disponíveis)
+
+  <details>
+    <p><summary>Abrir para visualizar os helpers disponíveis</summary>
```
Comment (wlsf82) [inline README_PT-BR.md:19]: Que tal assim?

```suggestion
    <p><summary>Abrir para visualizar todas as funções disponíveis</summary>
```

File: README_PT-BR.md [hunk 2]
```diff
@@ -98,6 +125,144 @@ Assim que for importado no seu arquivo de testes, você poderá utilizar os mét
 
 [Aqui você encontrará exemplos de uso de cada um dos métodos disponíveis.](docs/EXAMPLES.md#examples) (Importante: leia as notas no topo)
 
+## Helpers disponíveis
```
Comment (wlsf82) [inline README_PT-BR.md:128]:
```suggestion
## Funções disponíveis
```

File: README_PT-BR.md [hunk 3]
```diff
@@ -98,6 +125,144 @@ Assim que for importado no seu arquivo de testes, você poderá utilizar os mét
...
+
+Abaixo segue a lista de todos os helpers disponíveis no protractor-helper com seus respectivos exemplos:
```
Comment (wlsf82) [inline README_PT-BR.md:130]:
```suggestion
Abaixo segue a lista de todas as funções disponíveis no protractor-helper com seus respectivos exemplos:
```

File: README_PT-BR.md [hunk 4]
```diff
@@ -98,6 +125,144 @@ Assim que for importado no seu arquivo de testes, você poderá utilizar os mét
...
+
+### `setTimeout`
+
+Este método permite alterar o tempo de timeout de todos os métodos chamados após `setTimeout`.
```
Comment (wlsf82) [inline README_PT-BR.md:134]:
```suggestion
Este método permite alterar o _timeout_ de todos os métodos chamados após o `setTimeout`.
```

File: README_PT-BR.md [hunk 5]
```diff
@@ -98,6 +125,144 @@ Assim que for importado no seu arquivo de testes, você poderá utilizar os mét
...
+
+O `timeoutInMilliseconds` padrão é 5000 millisegundos
```
Comment (wlsf82) [inline README_PT-BR.md:136]:
```suggestion
O `timeoutInMilliseconds` padrão é 5000 milisegundos
```

File: README_PT-BR.md [hunk 6]
```diff
@@ -98,6 +125,144 @@ Assim que for importado no seu arquivo de testes, você poderá utilizar os mét
...
+
+Caso seja invocado sem que um parametro seja passado, o timeout será definido como o padrão.
```
Comment (wlsf82) [inline README_PT-BR.md:138]:
```suggestion
Caso seja invocado sem que um parametro seja passado, o _timeout_ será definido como o padrão.
```

File: README_PT-BR.md [hunk 7]
```diff
@@ -98,6 +125,144 @@ Assim que for importado no seu arquivo de testes, você poderá utilizar os mét
...
+
+> Um exemplo de uso desse método é a necessidade de assegurar que a tela seja carregada dentro de XX segundos, devido uma necessidade expressa pelo gerente de produtos. Isso ocorre frequentemente em sistemas de e-commerce emdatas especiais como a Blackfriday.
```
Comment (wlsf82) [inline README_PT-BR.md:140]:
```suggestion
> Um exemplo de uso desse método é a necessidade de assegurar que a tela seja carregada dentro de XX segundos, devido uma necessidade expressa pelo gerente de produtos. Isso ocorre frequentemente em sistemas de _e-commerce_ em datas especiais como a Black Friday.
```

File: README_PT-BR.md [hunk 8]
```diff
@@ -98,6 +125,144 @@ Assim que for importado no seu arquivo de testes, você poderá utilizar os mét
...
+
+[Exemplo](docs/EXAMPLES.md#settimeout)
+
+### `isCurrentUrlDifferentFromBaseUrl`
+
+Este método retorna um valor booleano dependendo se a url atual é diferente da url base. Não necessita de parâmetros.
```
Comment (wlsf82) [inline README_PT-BR.md:146]:
```suggestion
Este método retorna um valor booleano dependendo se a url atual é diferente da url base e não necessita de parâmetros.
```

File: README_PT-BR.md [hunk 9]
```diff
@@ -98,6 +125,144 @@ Assim que for importado no seu arquivo de testes, você poderá utilizar os mét
...
+
+### `waitForElementNotToBePresent`
+
+Este método é o oposto do anterior, de tal forma que espera até que um elemento não esteja presente no DOM.
+
+[Exemplo](docs/EXAMPLES.md#waitforelementnottobepresent)
+
+### `waitForElementVisibility`
+
+Este método aguarda até que um elemento esteja visível na página. Estar visível não significa apenas estar presente no DOM, mas também tem altura e largura superior a 0.
```
Comment (wlsf82) [inline README_PT-BR.md:166]:
```suggestion
Este método aguarda até que um elemento esteja visível na página. Estar visível significa não apenas estar presente no DOM, mas também ter altura e largura maior que 0.
```

File: README_PT-BR.md [hunk 10]
```diff
@@ -98,6 +125,144 @@ Assim que for importado no seu arquivo de testes, você poderá utilizar os mét
...
+
+### `waitForElementNotToBeVisible`
+
+Este método é o oposto do anterior, de tal forma que espera que um elemento não esteja visível na página. Por não visível, entenda-se que o elemento pode estar no DOM mas não estar visível.
+
+[Exemplo](docs/EXAMPLES.md#waitforelementnottobevisible)
+
+### `click`
+
+Este método é utilizado para efetuar um clique em um elemento assim que estiver em um estado viável e habilitado para o clique.
```
Comment (wlsf82) [inline README_PT-BR.md:178]:
```suggestion
Este método é utilizado para efetuar um clique em um elemento assim que estiver em um visível e habilitado para o clique.
```

File: README_PT-BR.md [hunk 11]
```diff
@@ -98,6 +125,144 @@ Assim que for importado no seu arquivo de testes, você poderá utilizar os mét
...
+
+### `clearFieldAndFillItWithText`
+
+Este método limpa o conteúdo de um campo de input assim que estiver visível, então preenche o mesmo com um texto.
+
+[Exemplo](docs/EXAMPLES.md#clearFieldAndFillItWithText)
+
+### `tap`
+
+Este método executa  a ação de toque em um elemento HTML capaz de eventos de toque/clique assim que estiver em um estado viável de toque/clique. É utilizado quando se está fazendo testes web mobile em emuladores, por exemplo.
```
Comment (wlsf82) [inline README_PT-BR.md:214]:
```suggestion
Este método executa  a ação de toque em um elemento HTML capaz de eventos de toque/clique assim que estiver em um estado possível de toque/clique. Tal método é utilizado quando se está fazendo testes web _mobile_ em emuladores, por exemplo.
```

File: README_PT-BR.md [hunk 12]
```diff
@@ -98,6 +125,144 @@ Assim que for importado no seu arquivo de testes, você poderá utilizar os mét
...
+
+### `waitForTextToBePresentInElement`
+
+Este método espera por um texto específico em um determinado elemento HTML.
+
+[Exemplo](docs/EXAMPLES.md#waitfortexttobepresentinelement)
+
+### `waitForTextNotToBePresentInElement`
+
+Este método é o oposto do anterior, de tal forma que espera que um texto não esteja presente em um determinado elemento do HTML.
```
Comment (wlsf82) [inline README_PT-BR.md:226]:
```suggestion
Este método é o oposto do anterior, de tal forma que espera que um texto não esteja presente em um determinado elemento HTML.
```

File: README_PT-BR.md [hunk 13]
```diff
@@ -98,6 +125,144 @@ Assim que for importado no seu arquivo de testes, você poderá utilizar os mét
...
+
+### `waitForUrlToBeEqualToExpectedUrl`
+
+Este método aguarda até que a URL seja igual a outra esperada. Este método é útil quando se quer executar ações em elementos apenas se estiver na URL correta.
+
+[Exemplo](docs/EXAMPLES.md#waitforurltobeequaltoexpectedurl)
+
+### `waitForUrlNotToBeEqualToExpectedUrl`
+
+Este método aguarda até que a URL não seja igual a uma a uma determinada URL. Este método é útil quando se quer continuar a executar ações em elementos apenas quando não estiver em uma URL específica.
```
Comment (wlsf82) [inline README_PT-BR.md:238]:
```suggestion
Este método aguarda até que a URL não seja igual a uma a uma outra determinada URL. Este método é útil quando se quer continuar a executar ações em elementos apenas quando não estiver em uma URL específica.
```

File: README_PT-BR.md [hunk 14]
```diff
@@ -98,6 +125,144 @@ Assim que for importado no seu arquivo de testes, você poderá utilizar os mét
...
+
+### `waitForUrlNotToContainString`
+
+Este método aguarda até que a URL não contenha um determinado texto. É utilizado quando se deseja fazer verificações baseadas na URL atual.
+
+[Exemplo](docs/EXAMPLES.md#waitforurlnottocontainstring)
+
+### `fillFieldWithTextAndPressEnter`
+
+Este método preenche um campo de input com um texto assim que estiver visível e então simula a ação de pressionar a tecla ENTER do teclado. É utilizado em casos como fazer uma busca e pressionar ENTER, em vez de preencher o campo e selecionar o botão de busca, por exemplo.
```
Comment (wlsf82) [inline README_PT-BR.md:256]:
```suggestion
Este método preenche um campo de input com um texto assim que estiver visível e então simula a ação de pressionar a tecla ENTER do teclado. É utilizado em casos como fazer uma busca e pressionar ENTER, em vez de preencher o campo e clicar no botão de busca, por exemplo.
```

File: README_PT-BR.md [hunk 15]
```diff
@@ -98,6 +125,144 @@ Assim que for importado no seu arquivo de testes, você poderá utilizar os mét
...
+
+[Exemplo](docs/EXAMPLES.md#fillFieldWithTextAndPressEnter)
+
+### `scrollToElement`
+
+Este método é usado para rolar a página até um elemento até que esteja visível no DOM.
```
Comment (wlsf82) [inline README_PT-BR.md:262]:
```suggestion
Este método é usado para rolar a página até um elemento até que tal elemento esteja visível no DOM.
```

File: README_PT-BR.md (hunk 10 above)
Comment (doamaral) [reply README_PT-BR.md:178]: Nesse acredito que a palavra "estado" foi removida indevidamente

Comment (wlsf82) [reply README_PT-BR.md:178]: Oops.

Comment (wlsf82) [reply README_PT-BR.md:178]: Vou ajustar a sugestão.

Comment (wlsf82) [reply README_PT-BR.md:178]: Done.

**Pull request merged**